bazel_versions.bzl

```diff
@@ -0,0 +1,11 @@
+"""Specifies the supported Bazel versions."""
+
+CURRENT_BAZEL_VERSION = "5.0.0"
+
+OTHER_BAZEL_VERSIONS = [
+    "4.2.2",
```

Comment:
We don't support below 5.0.0. Is there a way to add support for HEAD/rolling?

Reply:
I will remove `4.2.2`. There is not an easy way to specify HEAD/rolling. In other projects, I specify `6.0.0-pre.20220223.1` and periodically update it with whatever is the latest.